### PR TITLE
Issue #PS-1600 feat: API integration to show the list of events schedule against cohort

### PR DESCRIPTION
**Pull request by suvarnakale**
…ule against cohort

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
     "USER_CREDENTIAL_SEND_SUCCESSFULLY": "User Credentials Sent Successfully!",
     "CENTER_SESSIONS": "Center Sessions",
     "SCHEDULE_NEW": "Schedule New",
-    "UPCOMING_EXTRA_SESSION": "Upcoming Extra Sessions",
+    "UPCOMING_EXTRA_SESSION": "Upcoming Extra Sessions (Next 7 days)",
```
Comment (itsvick) [inline public/locales/en/common.json:108]: @suvarnakale this should be coming
 from configuration, accept variable  for 7 here

File: src/components/SessionCard.tsx [hunk 2]
```diff
@@ -30,6 +37,21 @@ const SessionsCard: React.FC<SessionsCardProps> = ({ data, children }) => {
     }
   };
 
+  useEffect(() => {
+    const result1 = convertUTCToIST(data?.startDateTime);
```
Comment (itsvick) [inline src/components/SessionCard.tsx:41]: @suvarnakale try to use meaningful variable name please

File: src/components/WeekCalender.tsx [hunk 3]
```diff
@@ -173,9 +175,28 @@ const Calendar: React.FC<any> = ({
...
+
+      rows.push(
+        <div className="row" key={endDay + ''}>
+          {days}
+        </div>
+      );
+    }
+
+    return (
+      <div className="body" style={{ width: showFromToday ? '90vw' : '100%' }}>
```
Comment (itsvick) [inline src/components/WeekCalender.tsx:196]: @suvarnakale why one is vw and one is in %?

File: src/components/SessionCard.tsx (hunk 2 above)
Comment (suvarnakale) [reply src/components/SessionCard.tsx:41]: ok

File: public/locales/en/common.json (hunk 1 above)
Comment (suvarnakale) [reply public/locales/en/common.json:108]: okay

File: src/components/WeekCalender.tsx (hunk 3 above)
Comment (suvarnakale) [reply src/components/WeekCalender.tsx:196]: becoz with % it takes width relative to its parent and here to not disturb its structure as it is used in dashboard as well. so given vw.

**Pull request merged**